test/unit/containers/slider-prompt.test.jsx

```diff
@@ -55,7 +55,7 @@ describe('Slider Prompt Container', () => {
         const componentProps = wrapper.find(SliderPromptComponent).props();
         componentProps.onChangeMin({target: {value: '1.0'}});
         componentProps.onOk();
-        expect(onOk).toBeCalledWith(1, 100, false);
+        expect(onOk).toHaveBeenCalledWith(1, 100, false);
```

Comment:
Would not be opposed to disabling this rule if we think it's annoying, it's just part of the recommended set. These changes were made just by using `--fix`, so pretty easy to deal with.